src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/payment_methods.html.twig

```diff
@@ -27,6 +27,23 @@
 
 {% block content %}
   <div class="container">
+
+    {% if paymentModules|length < 2 %}
+    <div class="card-block row">
+      <div class="card-text">
+        <div class="row">
+          <div class="col-sm">
+            <div class="alert alert-info" role="alert">
+              <div class="alert-text">
+                {{ 'We recommend providing at least two different payment methods. Only one payment method could be problematic if this option cannot be used by a customer because it will prevent him/her from ordering.'|trans({}, 'Admin.Payment.Notification') }}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
```suggestion
      <div class="card-block row">
        <div class="card-text">
          <div class="row">
            <div class="col-sm">
              <div class="alert alert-info" role="alert">
                <div class="alert-text">
                  {{ 'We recommend providing at least two different payment methods. Only one payment method could be problematic if this option cannot be used by a customer because it will prevent him/her from ordering.'|trans({}, 'Admin.Payment.Notification') }}
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
```
You should configure your IDE :wink: